src/archetypes/referencebrowserwidget/browser/popup.pt

```diff
@@ -97,6 +99,14 @@
                i18n:domain="plone"
                i18n:attributes="value label_search;"
                />
+  &nbsp;<span style="font-weight: bold; color: grey; border-bottom: 1px dotted; cursor: help;"
+              title="#"
+              id="searchWildcardHelp"
+              tal:condition="python: search_index in wildcardable_indexes"
+              i18n:attributes="title"
+              data-wildcardableIndexes="#"
+              tal:attributes='title python: widget.use_wildcard_search and "wild_card_search_enabled_help" or "wild_card_search_disabled_help";
+                              data-wildcardableIndexes wildcardable_indexes_as_json'>?</span>
```

Comment:
Hi @vincentfretin 

I updated the Pull request, now the help message translation is managed in the popup view code and there is a default provided.  This will be used in p.a.locales...

Could you please review, especially the proposed default translations and adapt it if necessary?

Thank you!

Gauthier